File: RedSheet.DbEntities/Models/vNanoScopeToNegotiate.cs

using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using Microsoft.EntityFrameworkCore;
using Rx.Core.Data.Attributes;
using RedSheet.DbEntities.Enums;

namespace RedSheet.DbEntities.Models
{
	[Table("vNanoScopeToNegotiates", Schema = "dbo")]
	[Serializable]
	public partial class vNanoScopeToNegotiate  
	{
 
	[Required] 
	[MaxLength(200)]
	public string ClassName { get; set; } 
 
	[DatabaseGenerated(DatabaseGeneratedOption.Identity)] 
	[Key]
	public int NanoScopeToNegotiateId { get; set; } 
 
	[Required] 
	[MaxLength(200)]
	public string NanoScopeToNegotiateName { get; set; } 

	public vNanoScopeToNegotiate()
	{
    }

    }
}

File: redsheet-portal/src/app/database-models/v-nano-scope-to-negotiate.ts
import { required, maxLength, range, nested } from '@rx/annotations';

export class vNanoScopeToNegotiate {
    constructor(vNanoScopeToNegotiate?: vNanoScopeToNegotiate )  {
        let properties = [ "className", "nanoScopeToNegotiateId", "nanoScopeToNegotiateName",];
        for (let property of properties)
            if (vNanoScopeToNegotiate && vNanoScopeToNegotiate[property])
                this[property] = vNanoScopeToNegotiate[property];
    }
 
    @required()
    @maxLength(200)
	className : string =   undefined;
 
	nanoScopeToNegotiateId : number =   0 ;
 
    @required()
    @maxLength(200)
	nanoScopeToNegotiateName : string =   undefined;


}
